.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and test
+
+on:
+  push:
+    branches: ['**', '!backport/**']
```

Comment:
also running unit tests on backports may provide some additional confidence against regressions, but I think you can also make a good case to exclude them.

Reply:
I am with Josh on having tests run on backports.